interfaces/builtin/system_power_control.go

```diff
@@ -0,0 +1,64 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const systemPowerControlConnectedPlugAppArmor = `
+# Description: Can reboot, power-off and halt the system.
+# Usage: reserved
+
+#include <abstractions/dbus-session-strict>
+#include <abstractions/dbus-strict>
+
+# gnome-session
+dbus (send)
+    bus=system
+    path=/org/freedesktop/systemd1
+    interface=org.freedesktop.systemd1.Manager
+    member={Reboot,PowerOff,Halt}
+    peer=(label=unconfined),
+`
+
+const systemPowerControlConnectedPlugSecComp = `
+# Description: Can inhibit and uninhibit screen savers in desktop sessions.
+# dbus
```

Comment:
Copy & pasted doc needs fixing. Also needs to fix variable names, and the commented line below should be removed if unnecessary.